charts/mattermost-rtcd/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: mattermost-rtcd
+description: A Helm chart for Kubernetes to deploy Mattermost's RTCD
+type: application
```

Comment:
Please add some more info. such as Maintainers. You can see example here -> https://github.com/mattermost/mattermost-helm/blob/master/charts/mattermost-enterprise-edition/Chart.yaml